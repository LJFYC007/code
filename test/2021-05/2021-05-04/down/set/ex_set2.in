50 200
3 187 30 70
97 104 56 139
137 195 4 188
64 148 9 122
105 157 8 125
58 78 9 185
13 87 13 176
79 119 15 121
82 149 16 51
102 110 50 126
65 138 13 66
77 84 7 99
161 164 3 126
89 119 54 165
88 184 167 181
46 194 89 109
2 83 77 196
131 151 141 166
60 196 140 149
122 135 25 33
126 152 91 197
80 194 13 141
35 142 56 142
56 109 65 127
2 110 53 159
5 15 4 102
68 153 114 124
94 198 88 123
20 33 92 200
136 159 101 176
8 165 56 169
105 154 93 166
100 117 132 184
33 187 152 161
81 129 58 97
43 93 128 184
71 167 54 175
21 107 9 169
59 167 49 188
55 161 115 162
133 193 71 165
29 99 16 182
57 103 6 198
6 162 41 45
89 158 27 193
85 100 69 92
132 199 32 129
39 179 74 99
117 168 55 146
54 75 9 81
20 27
20 25
5 10
26 35
20 23
8 31
24 34
9 34
19 45
14 43
45 48
9 20
29 39
20 28
10 46
8 19
8 19
34 39
10 11
16 41
10 33
25 25
5 9
36 46
3 29
28 32
35 36
22 34
30 50
4 7
15 30
9 16
9 21
10 22
41 45
15 33
11 31
37 49
5 17
22 23
12 39
36 46
9 23
15 49
40 45
28 33
5 43
32 50
38 47
10 24
27 45
27 40
17 17
4 10
3 25
10 40
12 16
24 43
18 38
13 19
39 39
41 45
9 26
18 21
33 40
26 45
28 29
18 35
9 43
23 28
43 44
7 20
16 34
37 45
13 33
36 49
28 39
8 44
15 23
8 24
43 43
34 36
23 35
32 46
14 34
17 30
35 49
17 20
7 27
42 49
39 39
2 25
1 9
9 10
1 2
49 50
13 36
37 39
46 49
20 28
32 39
34 36
21 28
17 23
23 31
44 46
24 49
32 36
19 47
32 44
28 41
1 27
43 44
46 49
31 42
12 50
12 50
11 21
2 2
22 33
33 34
30 46
18 46
23 29
6 32
50 50
49 49
24 41
29 43
6 37
2 31
43 44
34 45
22 40
16 19
41 41
18 48
17 42
47 49
31 42
27 39
30 34
36 37
1 5
17 31
35 36
30 36
27 48
10 22
30 40
30 37
4 38
40 46
29 38
36 48
28 42
13 15
3 28
14 24
23 35
28 35
25 28
9 23
42 43
10 14
48 50
23 47
16 50
6 40
14 19
15 40
7 27
12 39
11 14
22 24
9 15
23 29
5 15
2 24
12 41
10 33
46 46
50 50
14 26
5 23
10 37
36 39
4 22
3 11
20 29
27 50
16 25
17 21
12 43
24 34
28 48
22 41
41 42
26 40
13 21
