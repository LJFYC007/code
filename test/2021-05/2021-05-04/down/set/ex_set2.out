NO
122
50
101
122
NO
101
NO
NO
NO
69
NO
NO
NO
NO
NO
NO
58
50
NO
NO
2
16
71
NO
101
58
101
NO
13
NO
88
NO
NO
71
NO
NO
71
88
35
NO
71
NO
NO
71
101
NO
NO
71
NO
NO
NO
2
50
NO
NO
88
NO
NO
161
49
71
NO
131
NO
NO
92
NO
NO
88
6
NO
NO
71
NO
71
NO
NO
131
NO
6
58
101
NO
NO
NO
71
131
NO
69
49
NO
82
50
56
55
NO
54
69
NO
NO
58
91
131
101
69
NO
NO
NO
NO
NO
NO
6
69
NO
NO
NO
NO
56
101
100
NO
NO
92
NO
9
55
NO
NO
NO
NO
6
71
NO
131
71
NO
NO
55
NO
NO
101
54
56
NO
58
NO
NO
NO
NO
NO
NO
71
NO
71
NO
88
NO
NO
101
101
88
NO
16
54
55
NO
NO
NO
131
NO
NO
NO
54
56
88
92
88
NO
NO
NO
69
9
NO
NO
NO
54
NO
50
NO
NO
NO
131
NO
101
NO
NO
71
NO
NO
